src/GameModel.cpp

```diff
@@ -309,16 +310,16 @@ namespace gameModel{
     }
 
     auto Environment::getBallByID(const communication::messages::types::EntityId &id) const -> std::shared_ptr<Ball> {
-        if (this->quaffle->id == id) {
+        if (this->quaffle->getId()== id) {
             return this->quaffle;
         }
-        else if (this->snitch->id == id) {
+        else if (this->snitch->getId()== id) {
```

Comment:
ein Leerzeichen wäre schön ;)
```suggestion
        else if (this->snitch->getId() == id) {
```